packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,77 @@
+const Logger = require('../logger/logger');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+* Convenience method for getting the Terra test viewports by name.
+*
+* @param {string|string[]} sizes - the list of Terra viewport names.
+* @return {object} - Terra viewport sizes.
+*/
+const getViewports = (...sizes) => {
+  const { terraViewports } = global.browser.config;
+  let viewportSizes = Object.keys(terraViewports);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(size => terraViewports[size]);
+};
+
+/**
+* Sets the viewport for the test run to the Terra test viewport size.
+*
+* @param {string} [formFactor=huge] - the Terra test viewport.
+*/
+const setViewport = (formFactor) => {
+  const { terraViewports } = global.browser.config;
+  if (formFactor) {
+    const terraViewport = terraViewports[formFactor];
+    if (terraViewport !== undefined && typeof terraViewport === 'object') {
+      global.browser.setWindowSize(terraViewport.width, terraViewport.height);
+    } else {
+      throw logger.error(`The ${formFactor} formFactor supplied is not a viewport size supported by Terra.`);
+    }
+  } else {
+    const defaultViewport = terraViewports.huge;
```

Comment:
We should make sure to call this out in the documentation that huge is the default viewport size.

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/3cd30c44ecf7cee37cafa3e12b9c4b129f38fb5b